scripts/llnl/common_build_functions.py

```diff
@@ -71,11 +93,11 @@ def build_info():
     res["built_from_branch"] = "unknown"
     res["built_from_sha1"]   = "unknown"
     res["platform"] = get_platform()
-    rc, out = sexe('git branch -a | grep \"*\"',ret_output=True,error_prefix="WARNING:")
+    rc, out = shell_exec('git branch -a | grep \"*\"',ret_output=True,error_prefix="WARNING:")
```

Comment:
`ret_output` was changed to `return_output`

Reply:
thanks for catching this. fixed.